docs/examples/Buttons/Buttons.jsx

```diff
@@ -106,11 +135,105 @@ const code = `
 const properties = [
     {property: 'onClick', description: 'Function is called on button click', type: 'function'},
     {property: 'color', description: 'optional color of the button', type: 'string'},
-    {property: 'size', description: 'optional size (small, default, large)', type: 'string', default: 'default'},
+    {property: '(Inherited)', description: 'Ant design properties are inherited (see: https://ant.design/components/button/)'},
 ];
 
+// Example implementation
```

Comment:
Das zweite Example würde ich in eine separate Datei auslagern

Reply:
done